### DISCUSSION TITLE
git lfs replaced my docs with code!

### DISCUSSION THREAD

**Discussion by ghost** [git-lfs/git-lfs#5159] [Q&A]
I had a 70+ page document of fine arts and literature, and now it's replaced (along with all its cousins) with something like:
version https://git-lfs.github.com/spec/v1
oid sha256:60ed8149fd48e7b01f53be3d720596cfa59068557066910c6a03e29beb00b666
size 24925963

Halp!

I need my document recovered within two days!

**Comment by ghost** [comment]
Okay, looks like the command git` lfs pull `works the magic.
Had me worried! Gawd!

**Reply by bk2204** [reply]
Hey,

If you haven't run `git lfs install` already, then that's probably why this is occurring.  Once you've done that, any new checkouts or clones should automatically check out the large files.  Note that it doesn't affect checkouts that have already been made, so as you noted, then you'll need to use `git lfs pull` for that to take effect.
